### DISCUSSION TITLE
Controller account when switching to new infra/database

### DISCUSSION THREAD

**Discussion by WayneWayner** [subquery/subql#916] [Frontier TestNet - Bug Reporting]
![subquery_controller_acc](https://user-images.githubusercontent.com/60453944/165296296-c7b48a8a-ba1d-48f0-a8d7-2cfaa7638d70.png)

For season 2, I changed the infrastructure. I noticed that everything is already finished on the /account site. So it seems that nothing else needs to be configured. But the new infrastructure can't have got the private key of the controller account yet. 

Therefore it would be good if not only onchain is checked whether a controller account was defined, but also locally in the database and whether they match.

**Comment by mzxyz** [comment]
Thanks for reporting this issue, we have a new account management which is under development, that will fix this issue, thanks.
